sktime/tests/_config.py

```diff
@@ -332,6 +333,7 @@
     Imputer: {"method": "mean"},
     HampelFilter: {"window_length": 3},
     OptionalPassthrough: {"transformer": BoxCoxTransformer(), "passthrough": True},
+    FeatureSelection: {"method": "all"},
```

Comment:
please add this to `get_test_params` instead, new policy on where the defaults are 😄

Reply:
tests use `get_test_params` which (for downwards compatibility) looks in `_config`, but the primary location for the parameters is in `get_test_params` now